.buildkite/pipeline.yml

```diff
@@ -55,6 +55,35 @@ steps:
     if: build.pull_request.labels includes "benchmark"
     timeout_in_minutes: 30
 
+  - label: "AMDGPU - Julia 1.9 - No Artifacts"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: 1.9-nightly
+      - JuliaCI/julia-test#v1:
+      - JuliaCI/julia-coverage#v1:
+          codecov: true
+          dirs:
+            - src
+            - ext
+    agents:
+      queue: "juliagpu"
+      rocm: "*"
+      rocmgpu: "*"
+    command:
+      - julia -e """
+        using TOML
+        conf = TOML.parse(read(\"Project.toml\", String))
+        push!(conf[\"targets\"][\"test\"], \"AMDGPU\")
+        open(io -> TOML.print(io, conf), \"Project.toml\", \"w\")
+        """
```

Comment:
Buildkite complains about this but I don't know why: https://buildkite.com/julialang/nnlib-dot-jl/builds/729#0186358b-3599-4b7f-b98b-f7c0811b9cac/259-260.

Also, is a call to `Pkg.test()` required or will the test plugin handle that even if a `command` is specified?

Reply:
> Also, is a call to Pkg.test() required or will the test plugin handle that even if a command is specified?

I think no, removed it.

Reply:
> Buildkite complains about this but I don't know why

I think it was missing `;` at the end of lines. Added them.